### ISSUE TITLE
dynamodb delete-table fails

### ISSUE THREAD

**Issue by moritzzimmer** [localstack/localstack#2702]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description

Deletion of an existing table fails with an error when using `localstack/localstack-light` although the table is deleted. This behavior occurred using the aws cli (`aws-cli/2.0.26 Python/3.8.3 Darwin/19.5.0 botocore/2.0.0dev30`) as well as in Java unit tests using `testcontainers`.

## Expected behavior

Existing table is deleted without error.

## Actual behavior

Deleting the table fails with

```
An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Cannot do operations on a non-existent table
```

on the command line. Although the table is actually deleted.

# Steps to reproduce

## Command used to start LocalStack

```
docker run --rm -p 4566:4566 --env SERVICES=dynamodb localstack/localstack-light
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

create table

```
aws dynamodb create-table --table-name MusicCollection --attribute-definitions AttributeName=Artist,AttributeType=S AttributeName=SongTitle,AttributeType=S --key-schema AttributeName=Artist,KeyType=HASH AttributeName=SongTitle,KeyType=RANGE --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 --endpoint-url http://localhost:4566
```

verify

```
aws dynamodb list-tables --endpoint-url http://localhost:4566

{
    "TableNames": [
        "MusicCollection"
    ]
}
(END)
```

and try to delete

```
aws dynamodb delete-table --table-name MusicCollection --endpoint-url http://localhost:4566

An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Cannot do operations on a non-existent table
```

check again

```
aws dynamodb list-tables --endpoint-url http://localhost:4566

{
    "TableNames": []
}
(END)
```

**Comment by ninhkd** [comment]
Thanks @moritzzimmer. Please add lambda into service list so the command used to start LocalStack should be:
`docker run --rm -p 4566:4566 --env SERVICES=dynamodb,lambda localstack/localstack-light`

**Comment by whummer** [comment]
Thanks for reporting @moritzzimmer  - this should be fixed in #2715 . Can you please give it another try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks
